src/app/project-notifications/add-edit-project-notification/add-edit-project-notification.component.ts

```diff
@@ -67,59 +67,55 @@ export class AddEditProjectNotificationComponent implements OnInit, OnDestroy {
   ) { }
 
   public ngOnInit() {
+    this.regions = this.configService.lists.map(item => item.type === 'region');
```

Comment:
Same as the other changes, use `regions` directly from configService

Reply:
Fixed in d8e5e8e2